build/Targets/Settings.props

```diff
@@ -124,6 +124,7 @@
     <When Condition="'$(MSBuildProjectExtension)' == '.vbproj' OR '$(Language)' == 'VB' OR '$(ProjectLanguage)' == 'VB'">
       <PropertyGroup>
         <ProjectLanguage>VB</ProjectLanguage>
+        <LangVersion>Latest</LangVersion>
```

Comment:
👍 especially since we use a nuget package for the compiler and so it's possible for the IDE and the package to be out of date.  Until we can make the IDE understand what the latest version supported by the toolset package is, I think we should be explicit about it.